apps/builder/app/canvas/instance-hovering.ts

```diff
@@ -44,27 +45,41 @@ export const subscribeInstanceHovering = () => {
       hoveredInstanceSelectorStore.set(undefined);
       hoveredInstanceOutlineStore.set(undefined);
     }, 100);
+
+    // Fixes the bug, that new hover occures during timeout
```

Comment:
Not a bug if it is fixed

Reply:
? The code Fixes the bug
What the issue?